internal/scorecard/plugins/olm_tests.go

```diff
@@ -87,25 +111,6 @@ func NewCRDsHaveResourcesTest(conf OLMTestConfig) *CRDsHaveResourcesTest {
 	}
 }
 
-// AnnotationsContainExamplesTest is a scorecard test that verifies that the CSV contains examples via the alm-examples annotation
```

Comment:
Why this test was removed? Why the validation of the `alm-examples` as [here](https://github.com/dev4devs-com/postgresql-operator/blob/master/deploy/olm-catalog/postgresql-operator/0.1.2/postgresql-operator.v0.1.2.clusterserviceversion.yaml#L6) is no longer required?

Reply:
this is removed since csv validation catches that, the csv validation is performed by default when a CSV is read by the scorecard when it starts up.  In previous versions, the CSV was unmarshalled only, in this version the CSV is validated against the new validation API.  This is why the CSV alm-examples test was no longer necessary.